pkg/cmd/envInit_test.go

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package cmd
+
+import (
+	"testing"
+
+	"github.com/galasa.dev/cli/pkg/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCanCreateGalasaHomeFolderWhenNotAlreadyInitialised(t *testing.T) {
+	mockFileSystem := utils.NewMockFileSystem()
+	homeDir, _ := mockFileSystem.GetUserHomeDir()
+	galasaDir := homeDir + "/.galasa/"
+	m2Dir := homeDir + "/.m2/"
+	err := envInit(mockFileSystem)
+	if err != nil {
+		assert.Fail(t, err.Error())
+	}
+	assertBootstrapPropertiesCreated(t, mockFileSystem, galasaDir)
+	assertCPSPropertiesCreated(t, mockFileSystem, galasaDir)
+	assertCredentialsPropertiesCreated(t, mockFileSystem, galasaDir)
+	assertDSSPropertiesCreated(t, mockFileSystem, galasaDir)
+	assertOverridesPropertiesCreated(t, mockFileSystem, galasaDir)
+	assertSettingsXMLCreatedAndContentOk(t, mockFileSystem, m2Dir)
```

Comment:
You test all these other files, but not the .m2 ones you have introduced ?

Reply:
line 27 tests if settings.xml has been created which is in the m2 directory?

Reply:
oops my mistake